gradle/dependencies.gradle

```diff
@@ -5,19 +5,12 @@ ext {
     scalaInterfaceVersion = '2.11'
 
     commonVersions = [
-            finagle: '6.30.0',
-            guava: '16.0.1', // to match finagle
-            scrooge: '4.2.0',
-            twitterUtil: '6.29.0',
-            twitterServer: '1.15.0',
-            finatra: '2.1.1',
-            jackson: '2.4.4', // from finatra's build.sbt
-            ostrich: '9.13.0',
-            griddle: '1.7',
             slf4j: '1.7.12',
+            log4j: '1.2.17',
```

Comment:
Most probably there will be conflicts - I'm aiming to get a fat-jar ready solution so I'll bump into this for sure and will resolve. For now I'm assuming it's working as a standalone.